### PR TITLE
[feat] Add GPU hashtable and GPU workflow

### PR DESCRIPTION
**Pull request by rhdong**
- Add GPU hashtable
- Add workflow for GPU version release
- Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,31 +132,61 @@ pip install artifacts/tensorflow_recommenders_addons-*.whl
...
+| half  | CPU, GPU | - | CPU |
+| int32  | CPU, GPU | CPU | CPU |
+| int8  | CPU, GPU | - | CPU |
+| int64  | CPU | - | CPU |
+| double  | CPU, CPU | CPU |
+| bool  | - | - | CPU |
+| string  | CPU | - | - |
+
+##### To use GPU by `tfra.dynamic_embedding.Variable`
+The `tfra.dynamic_embedding.Variable` will ignore the device placement mechanism of TensorFlow,
```
Comment (Lifann) [inline README.md:164]: I think it should also be mentioned in [API annotation](https://github.com/tensorflow/recommenders-addons/blob/master/tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py#L164-L165) and [doc](https://github.com/tensorflow/recommenders-addons/blob/master/docs/api_docs/dynamic_embedding/Variable.md)

File: README.md [hunk 2]
```diff
@@ -138,31 +132,61 @@ pip install artifacts/tensorflow_recommenders_addons-*.whl
...
+
+##### Data Type Matrix for `tfra.dynamic_embedding.Variable` 
+|  Values \\ Keys  | int64  | int32 | string |
+|:----:|:----:|:----:|:----:| 
+| float  | CPU, GPU | CPU | CPU |
+| half  | CPU, GPU | - | CPU |
+| int32  | CPU, GPU | CPU | CPU |
+| int8  | CPU, GPU | - | CPU |
+| int64  | CPU | - | CPU |
+| double  | CPU, CPU | CPU |
```
Comment (Lifann) [inline README.md:159]: Missing a column.

File: README.md [hunk 3]
```diff
@@ -138,31 +132,61 @@ pip install artifacts/tensorflow_recommenders_addons-*.whl
...
-|:----------------------- |:---- |:---------| :------------ | :---- | :------------ |
-| tensorflow-recommenders-addons-0.1.0 | 2.4.1  | GCC 7.3.1 | 8.2.0 | 11.0 | [3.5, 5.2, 6.0, 6.1, 7.0, 7.5, 8.0] |
+de = tfra.dynamic_embedding.get_variable("VariableOnGpu",
+                                         devices=["/job:ps/task:0/GPU:0", ],
+                                         # ...
+                                         )
+```
 
-Check [nvidia-support-matrix](https://docs.nvidia.com/deeplearning/cudnn/support-matrix/index.html) for more details.
+**Use restrictions on GPU**
```
Comment (Lifann) [inline README.md:176]: Usage

File: README.md [hunk 4]
```diff
@@ -138,31 +132,61 @@ pip install artifacts/tensorflow_recommenders_addons-*.whl
...
+- Considering the size of the whl file, currently `dim` only supports less than or equal to 200, if you need longer `dim`, please submit an issue.
```
Comment (Lifann) [inline README.md:177]: Also mention in API and doc.

Comment (rhdong) [reply README.md:177]: accept

File: README.md (hunk 1 above)
Comment (rhdong) [reply README.md:164]: accept

**Review by Lifann — APPROVED** [review]
LGTM

**Pull request merged**